### PR TITLE
feat(onboarding): Update onboarding copy

### PR DESCRIPTION
**Pull request by ryan953**
Copy changes from @lizokm

Updated the header of the Quick start sidebar and removed the 'next up' section header (but not the other section heads)
Screen Shot 2022-11-11 at 11 41 01 AM

And then some copy changes inside a few tasks.

### CODE REVIEW

File: static/app/components/onboardingWizard/taskConfig.tsx [hunk 1]
```diff
@@ -142,7 +142,7 @@ export function getOnboardingTasks({
       task: OnboardingTaskKey.FIRST_EVENT,
       title: t('Capture your first error'),
       description: t(
-        "Time to test it out. Now that you've created a project, capture your first error. We've got an example you can fiddle with."
+        "Now that you've created a project, test it by capturing your first error. We've got an example you can fiddle with."
```
Comment (rahulchhabria) [inline static/app/components/onboardingWizard/taskConfig.tsx:145]: maybe just "Capture your first error."

File: static/app/components/onboardingWizard/taskConfig.tsx [hunk 2]
```diff
@@ -167,7 +167,7 @@ export function getOnboardingTasks({
       task: OnboardingTaskKey.INVITE_MEMBER,
       title: t('Invite your team'),
       description: t(
-        'Assign issues and comment on shared errors with coworkers so you always know who to blame when sh*t hits the fan.'
+        'Assign issues to the right people and comment back-and-forth on shared errors with coworkers.'
```
Comment (rahulchhabria) [inline static/app/components/onboardingWizard/taskConfig.tsx:170]: let's keep the original copy on this one. we get a decent amount of positive feedback for this tone and voice.

File: static/app/components/onboardingWizard/taskConfig.tsx [hunk 3]
```diff
@@ -179,7 +179,7 @@ export function getOnboardingTasks({
       task: OnboardingTaskKey.FIRST_INTEGRATION,
       title: t('Install any of our 40+ integrations'),
       description: t(
-        'Get alerted in Slack. Two-way sync issues between Sentry and Jira. Notify Sentry of releases from GitHub, Vercel, or Netlify.'
+        'Get alerts in Slack, sync Sentry and Jira, notify Sentry of releases from GitHub, Vercel, or Netlify.'
```
Comment (rahulchhabria) [inline static/app/components/onboardingWizard/taskConfig.tsx:182]: please keep the original. two-way sync is important.

File: static/app/components/onboardingWizard/taskConfig.tsx [hunk 4]
```diff
@@ -191,7 +191,7 @@ export function getOnboardingTasks({
       task: OnboardingTaskKey.SECOND_PLATFORM,
       title: t('Create another project'),
       description: t(
-        'Easy, right? Don’t stop at one. Set up another project and send it events to keep things running smoothly in both the frontend and backend.'
+        'Make it easy to quickly tell where errors are coming from by setting up separate Projects for your frontend and backend.'
```
Comment (rahulchhabria) [inline static/app/components/onboardingWizard/taskConfig.tsx:194]: please keep the original.

File: static/app/components/onboardingWizard/taskConfig.tsx [hunk 5]
```diff
@@ -289,7 +289,7 @@ export function getOnboardingTasks({
       task: OnboardingTaskKey.SOURCEMAPS,
       title: t('Upload source maps'),
       description: t(
-        "Deminify Javascript source code to debug with context. Seeing code in it's original form will help you debunk the ghosts of errors past."
+        'Make Javascript source code human readable and debug with greater context and speed. Seeing code in its original form will help you debunk the ghosts of errors past.'
```
Comment (rahulchhabria) [inline static/app/components/onboardingWizard/taskConfig.tsx:292]: please keep the original. this one reads a bit too market-y

**Comment by rahulchhabria** [comment]
@ryan953 - i left some comments and edits. there a few lines where i want to keep the originals. thank you.

File: static/app/components/onboardingWizard/taskConfig.tsx (hunk 1 above)
Comment (lizokm) [reply static/app/components/onboardingWizard/taskConfig.tsx:145]: How about: "Now that you've created a project, capture your first error. We've got...."  (If we just leave it as "Capture your first error." It'll be too repetitive with the title. :)

File: static/app/components/onboardingWizard/taskConfig.tsx (hunk 2 above)
Comment (lizokm) [reply static/app/components/onboardingWizard/taskConfig.tsx:170]: ok.

File: static/app/components/onboardingWizard/taskConfig.tsx (hunk 3 above)
Comment (lizokm) [reply static/app/components/onboardingWizard/taskConfig.tsx:182]: ok.

File: static/app/components/onboardingWizard/taskConfig.tsx (hunk 4 above)
Comment (lizokm) [reply static/app/components/onboardingWizard/taskConfig.tsx:194]: Obviously @rahulchhabria, you have a better sense of user sentiment here, but it seems to me that there's a key value prop missing for people who don't understand the benefit of setting up multiple projects, which is to be able to more quickly tell where exactly the error is coming from - the frontend/backend etc. I'm also wary of assuming we know how users feel. What if it wasn't easy for them for some reason? Then we've just made them feel bad. Just something to consider.

File: static/app/components/onboardingWizard/taskConfig.tsx (hunk 5 above)
Comment (lizokm) [reply static/app/components/onboardingWizard/taskConfig.tsx:292]: Definitely a learning for me here. I had to go read up on minification. But it looks like the proper term is unminify. Can we update to: "Unminify JavaScript source code..."

File: static/app/components/onboardingWizard/taskConfig.tsx (hunk 2 above)
Comment (lizokm) [inline static/app/components/onboardingWizard/taskConfig.tsx:170]:
```suggestion
        'Assign issues and comment on shared errors with coworkers so you always know who to blame when sh*t hits the fan.'
```
Let's leave as is per @rahulchhabria 's request.

File: static/app/components/onboardingWizard/taskConfig.tsx (hunk 5 above)
Comment (lizokm) [inline static/app/components/onboardingWizard/taskConfig.tsx:292]:
```suggestion
        'Unminify JavaScript source code and debug with context. Seeing code in its original form will help you debunk the ghosts of errors past.'
```

**Review by lizokm — COMMENTED** [review]
Waiting to hear back from @rahulchhabria on a few of these.

File: static/app/components/onboardingWizard/taskConfig.tsx (hunk 1 above)
Comment (rahulchhabria) [reply static/app/components/onboardingWizard/taskConfig.tsx:145]: 🤦 duh. sounds good. thank you.

File: static/app/components/onboardingWizard/taskConfig.tsx (hunk 3 above)
Comment (rahulchhabria) [reply static/app/components/onboardingWizard/taskConfig.tsx:182]: hm.. maybe. the sync is between sentry issues and jira issues. that's why it's important to call out. i'm not sure we gain much by changing it.

File: static/app/components/onboardingWizard/taskConfig.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (ryan953) [reply static/app/components/onboardingWizard/taskConfig.tsx:145]: Not accepting this based on the conversation above:
https://github.com/getsentry/sentry/pull/41279?no-redirect=1#discussion_r1020966868

> [lizokm](https://github.com/lizokm) [2 days ago](https://github.com/getsentry/sentry/pull/41279?no-redirect=1#discussion_r1020966868)
How about: "Now that you've created a project, capture your first error. We've got...." (If we just leave it as "Capture your first error." It'll be too repetitive with the title. :)

> @[rahulchhabria](https://github.com/rahulchhabria) rahulchhabria [22 hours ago](https://github.com/getsentry/sentry/pull/41279?no-redirect=1#discussion_r1022105782)
🤦 duh. sounds good. thank you.

File: static/app/components/onboardingWizard/taskConfig.tsx (hunk 3 above)
Comment (ryan953) [reply static/app/components/onboardingWizard/taskConfig.tsx:182]: For reference, this is how it looks in-app today:

Screen Shot 2022-11-15 at 11 45 34 AM

Comment (ryan953) [reply static/app/components/onboardingWizard/taskConfig.tsx:182]: I don't want to block on this point, so i'll also add here a suggestion to fully revert. So we can revert by clicking the button here, or resolve the conversation with the change to a single sentence.

```suggestion
        'Get alerted in Slack. Two-way sync issues between Sentry and Jira. Notify Sentry of releases from GitHub, Vercel, or Netlify.'
```

**Pull request closed without merging**